### ISSUE TITLE
can't find UIAccessibilityTraitHeader views in iOS9

### ISSUE THREAD

**Issue by WenxingYan** [kif-framework/KIF#720]
All section headers tests are failing. Anyone in the same situation? Or anyone not seeing this but using UIAccessibilityTraitHeader?

**Comment by phatmann** [comment]
@WenxingYan, did these tests all pass in iOS 8?

**Comment by sumitagarwal** [comment]
I am having the same issue. The tests pass in iOS 8 and Xcode 6 however they all fail in Xcode 7 and iOS 9.

**Comment by WenxingYan** [comment]
Yes they all passed previously.

**Comment by phatmann** [comment]
Can you post a code snippet with the failing test?
